### ISSUE TITLE
Replace all hard coded urls with reverse lookups

### ISSUE THREAD

**Issue by martsberger** [NorthBridge/alliance-community#38]
It's much cleaner and easier to maintain the app if urls are looked up from the url configuration, rather than hard coded. This means if you want a url like:

```
/alliance/apps/backlog/
```

Then in your python code you do:
```
reverse('backlogs')
```

And in a template you would have:
```
{% url 'backlogs' %}
```

Where 'backlogs' is the name of the url (give in the urls.py url configuration file).

The django docs explain it all:
https://docs.djangoproject.com/en/1.8/topics/http/urls/#reverse-resolution-of-urls

**Comment by hectron** [comment]
@martsberger can you please take a look at this?

**Comment by hectron** [comment]
This was resolved with Pull Request #55 .
